functests/setup_teardown.go

```diff
@@ -35,8 +35,6 @@ func AfterTestSuiteCleanup() {
 	err = t.DeleteNamespaceAndWait(TestNamespace)
 	gomega.Expect(err).To(gomega.BeNil())
 
-	// TODO uninstall storage cluster.
-	// Right now uninstall doesn't work. Once uninstall functions
-	// properly, we'll want to uninstall the storage cluster after
-	// the testsuite completes
+	err = t.UninstallOCS(ocsRegistryImage, localStorageRegistryImage)
```

Comment:
Here's a thought.

The majority of the functest execution time is spent setting up the environment. So when devs are writing and executing their tests locally, it's likely they'd like to keep the storagecluster online to avoid the setup time.

Can we figure out a way to opt in to the uninstall step?  CI should of course always uninstall to validate that flow, but it's possible during development when a dev is focused on their single test that they'd like to keep the storage cluster around.

Reply:
I agree, that would be handy.
Added a flag to disable the uninstall operation during functests. eg.
OCS_CLUSTER_UNINSTALL=false make functest

Defaults to true.